spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/s3/properties/S3Properties.java

```diff
@@ -93,6 +94,25 @@ public class S3Properties extends AwsClientProperties {
 	@NestedConfigurationProperty
 	private S3CrtClientProperties crt;
 
+	/**
+	 * Enables S3 integration for spring.config.import.
+	 */
+	private boolean enableImport = true;
+
+	/**
+	 * Properties related to configuration reload.
+	 */
+	@NestedConfigurationProperty
+	private ReloadProperties reload = new ReloadProperties();
```

Comment:
i think these two properties should not be at the same level as the rest as they apply specifically to config import feature. Perhaps we can move them to inner class `Config`? What do you think?

Reply:
I think this make sense. Lets confirm so that I can address this comment.